### PR TITLE
build: make installable via poetry / pipx

### PR DESCRIPTION
**Pull request by madlabman**

### CODE REVIEW

**Review by mymphe — APPROVED** [review]
Awesome feature!

File: pyproject.toml [hunk 1]
```diff
@@ -1,22 +1,25 @@
 [tool.poetry]
-name = "explorer-github-diff-checker"
-version = "0.2.0"
+name = "diffyscan"
+version = "0.1.0"
```
Comment (TheDZhon) [inline pyproject.toml:3]: Sorry, but if we want to package it, then I would plug the automated versioned builds here:

```toml
version = "0.0.0"

# ...

[tool.poetry-dynamic-versioning]
enable = true
vcs = "git"
style = "semver"
metadata = false

[build-system]
requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.1.0,<2.0.0"]
build-backend = "poetry_dynamic_versioning.backend"
```

Comment (TheDZhon) [reply pyproject.toml:3]: Also, please consider adding a command line argument to retrieve the version upon launch.

**Review by TheDZhon — CHANGES_REQUESTED** [review]
Thank you for the great proposal!

I'd suggest addressing only a single suggestion regarding the version bookkeeping approach to make it more robust.

**Review by skhomuti — APPROVED** [review]

**Review by TheDZhon — APPROVED** [review]
Awesome work, merging to the upstream :+1:

**Pull request merged**